packages/api/src/controllers/playback.ts

```diff
@@ -195,7 +209,7 @@ async function getPlaybackInfo(
   isCrossUserQuery: boolean,
   origin: string,
   withRecordings?: boolean,
-  recordCatalystObjectStoreId?: string
+  req?: Request
```

Comment:
This is duplicated, the request is already the first argument to this function.

Either extract more fields from that argument, or replace the deconstruction syntax with grabbing the whole `req` value and use it in the implementation.